lib/ardb/cli.rb

```diff
@@ -5,8 +5,18 @@ module Ardb
 
   class CLI
 
-    class InvalidCommand; end
+    class InvalidCommand;            end
+    class ConnectCommand;            end
+    class CreateCommand;             end
+    class DropCommand;               end
+    class MigrateCommand;            end
+    class GenerateMigrationsCommand; end
     COMMANDS = Hash.new{ |h, k| InvalidCommand.new(k) }.tap do |h|
+      h['connect']             = ConnectCommand
+      h['create']              = CreateCommand
+      h['drop']                = DropCommand
+      h['migrate']             = MigrateCommand
+      h['generate-migrations'] = GenerateMigrationsCommand
```

Comment:
@kellyredding - I think you want `GenerateMigrationCommand` here (migrations is plural instead of singular, double check me but I think that's what you have defined later in the file). I also think you want the command to be singular as it only generates a single migration file.